lib/css/modules/_nightMode.scss

```diff
@@ -1382,3 +1382,19 @@
 		}
 	}
 }
+
+
+.res-nightmode:not(.res-nightMode-coloredLinks){
```

Comment:
Does nightmode options include a colored links boolean body class option yet? Should probably be enabled by default.